curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-higher-order-arrow-functions.english.md

```diff
@@ -71,13 +106,18 @@ console.log(squaredIntegers);
 <section id='solution'>
 
 ```js
```

Comment:
This solution looks good, but could you remove the console.log at the very end?:

```js
const squareList = (arr) => {
  "use strict";
  const positiveIntegers = arr.filter((num) => {
    return num >= 0 && Number.isInteger(num);
      // add code here
  });
  const squaredIntegers = positiveIntegers.map((num) => {
    // add code here
    return num ** 2;
  });
  // add code here
  return squaredIntegers;
};
// test your code
const squaredIntegers = squareList(realNumberArray);
```

Reply:
I'll need you to take a look at the newest commit. For some reason the tests are not passing and I cannot figure out why.